### ISSUE TITLE
[Feature]:  alphabetical/numerical sorting

### ISSUE THREAD

**Issue by calimbrasti94** [dromse/obsidian-gamified-tasks#44]
### What kind of feature do you want?

Hi, great plugin, thanks! It would be nice to have the possibility to sort/order (date, name, idk..) the tasks, in the dedicated tab.

### Additional questions

- [ ] I want to work on this feature request

**Comment by dromse** [comment]
Hi @calimbrasti94,

Thank you for the feedback and for using my plugin! I'm glad you find it useful. The idea to sort/order tasks by date, name, or other criteria is excellent. Just to clarify, are you looking for both ascending and descending order options? Additionally, when you mention sorting by date, do you mean sorting completed tasks? Regarding sorting by name, could you please provide more details? If it's about searching by name, that feature is already implemented. Let me know if you're looking for something different.

**Comment by calimbrasti94** [comment]
Hi @dromse,

I don't really have a clear idea, but I can tell you that the system I found convenient with which to integrate your plugin is the reward and keep track of the documents I need to study. Using it a little bit I found it difficult to see at a glance what and how many tasks I had created. now, the tasks for learning materials all have the same task syntax [ example: `- [ ] file_name #count/0/5 #every/day #diff/trivial` ], with the name varying according to the file name, while the others, which maybe I had created in the meantime, but were not belonging to the same “category” for study, were being swamped by the ones created en masse. i don't know, maybe a simple alphabetical/numerical order, either ascending or descending, might be enough already.

p.s.: Thanks for asking and I hope I was helpful. Fantastic plugin ! Let's gamify Obsidian to the max !

p.p.s. not requested: when I read the description of the plugin, I don't deny that I imagined every task executed as an accomplished quest and that some kind of minecraft-style puppet was doing something ⚔️😅 

Translated with DeepL.com (free version)
